### ISSUE TITLE
Fatal Python error: deallocating None - lastupdate with unknown values

### ISSUE THREAD

**Issue by pan-ellox** [commx/python-rrdtool#21]
Hello,

Regarding python 2/3 version, using rrdtool crashes my script. Here you can find isolated case:

```python

import rrdtool

RRD_PATH = 'database.rrd'

if __name__ == '__main__':
    rrdtool.create(
        RRD_PATH,
        "--step", "5",
        'DS:temp1:GAUGE:300:U:U',
        'DS:temp2:GAUGE:300:U:U',
        'RRA:AVERAGE:0.5:1:60',
    )

    rrdtool.update(RRD_PATH, 'N:22.3:U')

    i = 0
    while True:
        print(i)
        print(rrdtool.lastupdate(RRD_PATH))

        i += 1
        if i > 100000:
            break
```

At some iteration it throws:
```python
2062
{'ds': {'temp1': 22.3, 'temp2': None}, 'date': datetime.datetime(2017, 11, 17, 18, 35, 38)}
Fatal Python error: deallocating None
```

With making more inputs in database (e.g. instead 1 unknown, make 4 unknowns) it throws ~4 times quicker.

Thanks,

Maciej

**Comment by commx** [comment]
This issue has been fixed in v0.1.12 which is now available both as release on Github and via PyPI.

Thanks for reporting!

**Comment by pan-ellox** [comment]
Awesome, thanks for an edit!
